src/components/label/label.jsx

```diff
@@ -3,15 +3,27 @@ import {Localize} from '../../lib/localize';
 
 const lookup = new Localize().lookup;
 
+function replaceAll(str, mapObj){
+	const keys = Object.keys(mapObj);
+	for (const key of keys) {
+		str = str.replace(key, mapObj[key]);
+	}
+	return str;
+}
+
 export default class Label extends Component {
 	static defaultProps = {
 		prefix: ''
 	};
 
 	render(props, state) {
-		const { prefix, localizeKey, className, children } = props;
+		const { prefix, localizeKey, className, replacements, children } = props;
 		const key = prefix ? `${prefix}.${localizeKey}` : localizeKey;
-		const localizedContent = lookup(key);
+		let localizedContent = lookup(key);
```

Comment:
Could be `const localizedContent  = replacements ? replaceAll(localizedContent, replacements) : lookup(key);`